.buildkite/test-pipeline.yaml

```diff
@@ -319,6 +319,17 @@ steps:
     - export VLLM_WORKER_MULTIPROC_METHOD=spawn
     - pytest -v -s -x lora/test_long_context.py
 
+- label: Weight Loading Multiple GPU Test
+  working_dir: "/vllm-workspace/tests"
+  num_gpus: 2
+  source_file_dependencies:
+  - vllm/
+  - tests/weight_loading
+  commands:
+    - export VLLM_WORKER_MULTIPROC_METHOD=spawn
```

Comment:
you can drop this env variable

Reply:
it was complaining about it before but can double check and remove